Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -55,14 +55,16 @@
 - type: entity
   parent: ClothingEyesBase
   id: ClothingEyesGlassesMeson
-  name: engineering goggles #less confusion
-  description: Green-tinted goggles using a proprietary polymer that provides protection from eye damage of all types.
+  name: mesons #less confusion
+  description: Green-tinted goggles that use a proprietary material that provides protection from eye damage and allows you to see through walls.
   components:
   - type: Sprite
     sprite: Clothing/Eyes/Glasses/meson.rsi
   - type: Clothing
     sprite: Clothing/Eyes/Glasses/meson.rsi
   - type: EyeProtection
```

Comment:
Perhaps outside the scope of this PR, but now that these are mesons they should stop being welding goggles so engineers actually use welding helmets (or goggles if later added)

Reply:
this is a balance issue and someone can fix it in the ~~~future~~~ if it ends up being a problem worth addressing